packages/eslint-plugin/lib/config/recommended.js

```diff
@@ -182,5 +182,6 @@ module.exports = {
     '@department-of-veterans-affairs/correct-apostrophe': 1,
     '@department-of-veterans-affairs/cypress-viewport-deprecated': 1,
     '@department-of-veterans-affairs/prefer-web-component-library': 1,
+    '@department-of-veterans-affairs/design-system': 1,
```

Comment:
Can we make the rule name a bit more specific? Maybe something like `deprecated-classes`, or even very specific like `deprecate-primary-button`.

Reply:
The intent here was to create a more generic rule name that we could reuse for non web component rules, but I can change it if that's not what we want.

Reply:
`deprecated-classes` sounds like it could be re-used for non web component rules unless I'm misunderstanding things.